.changeset/red-cheetahs-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-octopus-deploy': minor
+---
+
+Added support for octopus deploy spaces.
```

Comment:
Might want to mention in just a short sentence what annotation changes you made, helping users adapt accordingly. Also mention that the arguments to some exported things are now a single object instead of separate args.